### PR TITLE
Decam (but actually NSC)

### PR DESCRIPTION
**Pull request by SunilSimha**
The name is no longer relevant because the main DECam analysis module has been moved to `FLIMFLAM`. Instead, this contains the module to slurp data from the NOIRLab source catalog and some minor EAZY fixes.

### CODE REVIEW

File: frb/galaxies/eazy.py [hunk 1]
```diff
@@ -132,6 +132,8 @@ def eazy_input_files(photom, input_dir, name, out_dir, prior_filter=None,
             Name of the source being analzyed
         out_dir (str):
             Path to eazy OUTPUT folder *relative* to the input_dir
+        id_col (str):
```
Comment (profxj) [inline frb/galaxies/eazy.py:135]: add ", optional"

File: frb/galaxies/eazy.py [hunk 2]
```diff
@@ -166,6 +168,10 @@ def eazy_input_files(photom, input_dir, name, out_dir, prior_filter=None,
     if cosmo is None:
         cosmo = Planck15
 
+    # Convert to table if input photom is a dict.
+    if type(photom)==dict:
```
Comment (profxj) [inline frb/galaxies/eazy.py:172]: I'd start using `isinstance`

I think it is preferred, e.g.

`if isinstance(photom, dict):`

Comment (profxj) [reply frb/galaxies/eazy.py:172]: although in this case, I think you can just do:

`photom = Table(photom)` as a Table will recast to itself..

File: frb/galaxies/eazy.py [hunk 3]
```diff
@@ -184,10 +190,20 @@ def eazy_input_files(photom, input_dir, name, out_dir, prior_filter=None,
     # Test combo
     assert combo in _acceptable_combos, "Allowed values of 'combo' are {}".format(_acceptable_combos)
 
+    # Test id col
+    if id_col is None:
+        if 'id' in photom.colnames:
+            id_col = "id"
```
Comment (profxj) [inline frb/galaxies/eazy.py:196]: you can pull `id_col = "id"` out of the inner if/else block
as you set it as such in both cases.

**Review by profxj — CHANGES_REQUESTED** [review]
Please add a unit test or two (that will only be run locally, I suspect, although
you could try adding NOIR lab back to the CI).

Thanks!

File: frb/galaxies/eazy.py (hunk 2 above)
Comment (SunilSimha) [reply frb/galaxies/eazy.py]: Changed.

File: frb/galaxies/eazy.py (hunk 1 above)
Comment (SunilSimha) [reply frb/galaxies/eazy.py]: Added.

File: frb/galaxies/eazy.py (hunk 3 above)
Comment (SunilSimha) [reply frb/galaxies/eazy.py]: Done.

**Review by profxj — APPROVED** [review]
Looks good!

Handle conflicts and merge.

**Pull request merged**